public-api/src/main/resources/public-api.yaml

```diff
@@ -353,6 +353,26 @@ definitions:
             $ref: "#/definitions/Concept"
           description: standard concepts associated with the matched concept
 
+  ConceptSynonym:
```

Comment:
Not seeing where this gets used?

Do we want full objects representing the synonyms on the client, or just an array of strings like above?

Reply:
We just need the array of strings with the concept. This is just the intermediary object needed for the tests.